EIPS/eip-1193.md

```diff
@@ -314,18 +323,24 @@ The Provider **MAY** expose methods and properties not specified in this documen
 
 #### request
 
+> The `request` method is intended as a transport- and protocol-agnostic wrapper function for Remote Procedure Calls (RPCs).
+
 ```typescript
-type RequestParams = Array<any> | { [key: string]: any };
+interface RequestArguments {
+  method: string;
+  params?: Array<any> | { [key: string]: any };
```

Comment:
Same comments as above about `any`.  Always prefer `unknown`.

Reply:
My bad for not clarifying: all references to `any` were changed to `unknown` in 7753ce8